.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: "\U0001F41B Bug Report"
```

Comment:
It's odd that this emoji is not rendered but the one below is. Maybe we should double-check it renders correctly after merging

Reply:
That's because it's not an emoji character in the file, it's a Unicode escape sequence in the string literal. I don't know why our existing issue templates did it that way but either Unicode escapes or actual emoji characters seem to work.